README.md

```diff
@@ -79,7 +79,9 @@ After building the [TF-M regression](#Building-the-TF-M-Regression-Test) or
 [PSA compliance tests](#Building-the-PSA-Compliance-Test) for the target, it should be
 followed by building a Mbed OS application. This will execute the test suites previously built.
 
-Configure appropriate test in the `config` section of `mbed_app.json`.
+Configure appropriate test in the `config` section of `mbed_app.json`. If you want to
```

Comment:
Configure an appropriate test in the `config` section of `mbed_app.json`. If you want to
flash and run tests manually, please ensure `wait-for-sync` is 0 so that tests start without
waiting.